AdyenUI/AdyenTheme.swift

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+// A shared instance of the  theme for easy access throughout your SDK
+public class AdyenTheme {
+    
+    public var currentColorScheme: ColorScheme = .default
+    public var currentFontStyle: FontStyle = .default
+
+    // Available styles
+    public var buttonStyle = AdyenButtonTypes()
```

Comment:
style != types, this doesn't read nicely
next line is consistent with style = style

Reply:
I have renamed `AdyenButtonTypes` to `AdyenButtonStyles`.